pipdeptree.py

```diff
@@ -339,6 +341,52 @@ def jsonify_tree(tree, indent):
                       indent=indent)
 
 
+def dump_graphviz(tree, output_format='dot'):
+    """Output dependency graph as one of the supported GraphViz output formats.
+
+    :param dict tree: dependency graph
+    :param string output_format: output format
+    :returns: representation of tree in the specified output format
+    :rtype: str or binary representation depending on the output format
+
+    """
+    try:
+        from graphviz import backend, Digraph
+    except ImportError:
+        print('graphviz is not available, but necessary for the output '
+              'option. Please install it.', file=sys.stderr)
+        sys.exit(1)
+
+    if output_format not in backend.FORMATS:
+        print('%s is no supported output format.' % output_format,
```

Comment:
Can you please use str.format function for string interpolation for consistency? (I know the % syntax is used in a couple of places but I am planning to replace it with str.format)